README.md

```diff
@@ -45,15 +45,109 @@ This demo contains below applications.
     • Node.js Web application https://github.com/redhatHameed/3ScaleFuseAMQ/tree/master/nodejsalert-ui
     • 3scale (Openshift on-premises) environment
 
-### Automation of Applications in OpenShift.
+## Automation of Applications in OpenShift
+### Build and deploy with pipelines
+***The application works under [OpenShift](https://www.okd.io/) or [Minishift](https://www.okd.io/minishift/)***
 
-TODO: -The below deployment will be achieved using Jenkins Pipelines CICD approach.
+The following instructions assuming that you are using OpenShift. But the same could be applied to Minishift as well.
 
-     • Deploying application to DEV Environment.
-     • Deploying application from DEV to UAT Environment.
-     • Deploying application in Prod Environment.
-     • Deploying application to Prod DR Environment.
-     • Publishing API’s in 3scale : (By using Operator or ansible playbooks).
+Download the source codes from git repository by either forking, or simply cloning it. 
 
+```
+git clone https://github.com/RHsyseng/IntegrationApp-Automation.git  
+```
+Assume you have OpenShift cluster ready and running.
 
+```
+# login to the OpenShift
+oc login <OpenShift cluster url> --token=<OpenShift user login token>
+
+```
+
+Setup `rh-dev`, `rh-test` and `rh-prod` OpenShift projects as the target environment (you may skip this step if you already have the environment ready, and this script will first delete the original projects in OpenShift and create the new ones).
+    
+```
+./setup/setup.sh
+```
+If you ran the above setup.sh, the pipelines are automatically imported. In case you want to further customize the pipelines, please following the instructions below.
```

Comment:
If you **run**
, please **follow** the instructions below

Reply:
edited, and pushed.